neuralforecast/common/_scalers.py

```diff
@@ -9,48 +9,50 @@
 import torch.nn as nn
 
 # %% ../../nbs/common.scalers.ipynb 7
-def masked_median(x, mask, dim=1, keepdim=True):
+def masked_median(x, mask, dim=-1, keepdim=True):
```

Comment:
Ibid. It is unclear that the autoregresive features channel is what we want to normalize.

Reply:
This is only the default, set to -1 to follow PyTorch conventions. Different base classes have the time in different dimensions of the tensor